featuremanagement/_featurefilters.py

```diff
@@ -12,36 +23,36 @@ class FeatureFilter(ABC):
     """
 
     @abstractmethod
-    def evaluate(self, context, **kwargs):
+    def evaluate(self, context: Mapping, **kwargs: Unpack[RequestParams]) -> bool:
         """
         Determine if the feature flag is enabled for the given context.
 
         :param Mapping context: Context for the feature flag.
         """
 
     @property
-    def name(self):
+    def name(self) -> str:
         """
         Get the name of the filter.
 
         :return: Name of the filter, or alias if it exists.
         :rtype: str
         """
         if hasattr(self, "_alias"):
-            return self._alias
+            return self._alias  # type: ignore
```

Comment:
I don't understand the comment "type: ignore" here.

Reply:
This comment and those like it are things that the type hints for Mypy are not correct yet. This one in particular I haven't figured out how to get it to work correctly. So right now I'm suppressing the error with this.